app/scripts/util.js

```diff
@@ -85,6 +103,7 @@ var util = {
 		}
 		page.redirect('/error');
 	},
+	emailRegex: '[a-z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&\'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?',
```

Comment:
`pattern`, not `allowedPattern`.
Polymer docs - `A pattern to validate the input with.`